storage/storage.go

```diff
@@ -1137,24 +1137,21 @@ func (o *ObjectAttrs) toRawObject(bucket string) *raw.Object {
 
 // toProtoObject copies the editable attributes from o to the proto library's Object type.
 func (o *ObjectAttrs) toProtoObject(b string) *storagepb.Object {
-	var ret *timestamppb.Timestamp
-	if !o.RetentionExpirationTime.IsZero() {
-		ret = timestamppb.New(o.RetentionExpirationTime)
-	}
-	var ct *timestamppb.Timestamp
-	if !o.CustomTime.IsZero() {
```

Comment:
What happened here? We definitely need to be able to get/set CustomTime.

Reply:
I moved them down to use an inline helper `toProtoTimestamp`